src/main/java/io/confluent/connect/hdfs/wal/WALConstants.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2015 Confluent Inc.
```

Comment:
nit: If you apply any changes based on the comment above, you may change this one too to be `2017` instead. Otherwise, never mind.